spec/src/main/asciidoc/architecture.asciidoc

```diff
@@ -30,15 +30,15 @@ Only this Specification will be generated and maintained to document the content
 
 In reverse chronological order, here's the specification for each MicroProfile release.
 
-[[microprofile2.0]]
-=== MicroProfile 2.0 (2Q2018)
+[[microprofile2.0.1]]
+=== MicroProfile 2.0.1 (3Q2018)
 
-MicroProfile 2.0 is the sixth release for the https://projects.eclipse.org/projects/technology.microprofile[Eclipse MicroProfile project].
+MicroProfile 2.0.1 [1] is the sixth release for the https://projects.eclipse.org/projects/technology.microprofile[Eclipse MicroProfile project].
 This is a major new release for MicroProfile since the Java EE dependencies are now based on Java EE 8.
```

Comment:
This should say this is a micro release for MicroProfile, which is to fix the incorrect maven dependency on JAX-B

Reply:
I like this "micro release" terminology.  I'll work this in...  Thanks.